esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/install_cold/bin/install_cold.sh

```diff
@@ -3,6 +3,17 @@
 BASH_XTRACEFD=1
 set -eux
 
+# Remove esmvaltool and esmvalcore directories, if they exist.
+if [[ -d ${ESMVALTOOL_DIR} ]]; then
+    rm -rf "${ESMVALTOOL_DIR}"
+fi
+
+if [[ -d ${ESMVALCORE_DIR} ]]; then
+    rm -rf "${ESMVALCORE_DIR}"
+fi
```

Comment:
Would it be possible to move this bit of code so that it is closer to the code that requires it, please? (i.e. after the `rtw-env` section and just before the cloning section.)

Reply:
[71e9256](https://github.com/ESMValGroup/ESMValTool/pull/2803/commits/71e925628af88ad9120e9942b79316ab7c1a2c56)